modules/planning/tasks/deciders/speed_bounds_decider/speed_bounds_decider.cc

```diff
@@ -31,6 +31,7 @@
 #include "modules/planning/tasks/deciders/speed_bounds_decider/speed_limit_decider.h"
 #include "modules/planning/tasks/deciders/speed_bounds_decider/st_boundary_mapper.h"
 
+#define ADEBUG AERROR
```

Comment:
This is for debugging purpose only. Should remove this line in the commit because otherwise the DreamLand will fail due to memory limits.

Reply:
Ok